testdata/script/linker.txtar

```diff
@@ -1,3 +1,5 @@
+# Past garble versions might not properly patch cmd/link with "git apply"
```

Comment:
stealth edits :)

Reply:
Ah yeah. I remembered your previous commit and I thought some docs with context would help.